windlass/windlass.py

```diff
@@ -167,9 +167,10 @@ def main():
     try:
         g.run(
             process,
-            ns=ns,
             artifact_name=ns.artifact_name,
             parallel=not ns.no_parallel,
+            # following args are for the process function
+            ns=ns,
```

Comment:
I left comment in commit instead of in PR - but it seems that ns is keyword passed by this patch and not used subsequently?

Reply:
I simply moved it down from two lines above, and the comment is indicating that this and two arguments below are only used at https://github.com/HewlettPackard/windlass/pull/13/files#diff-6680089b2fb87c59a2b376bbeb3f4720L27-L48

Reply:
I did not notice it was moved :-(